src/__tests__/index.spec.ts

```diff
@@ -25,7 +25,7 @@ describe('request', () => {
         restoreMockFetch();
     });
 
-    it('makes request to API base url default w/ no token when no configuration is specified', async () => {
+    it('makes request to API base url default w/ no token when no configuration is specified', async() => {
```

Comment:
the eslint rule wants to get _rid_ of the space? ugh

Reply:
I think @jonathancreamer-eb called me out on this in brite-cli as the no space is expected.